### DISCUSSION TITLE
Confusing: unclassified publications are shown as "miscellaneous" in front-office

### DISCUSSION THREAD

**Discussion by mietcls** [ugent-library/biblio-backend#1199] [General]
# Bug description
Confusing: unclassified publications are shown as "miscellaneous" in front-office.

Edit: To Do Miet: Figure out whether "Classification pending" is a good port with Biblio team.

## Steps to Reproduce

1. Find an unclassified record in the back-office
2. View it in the front-office

Example of an A1 publication
![image](https://github.com/ugent-library/biblio-backoffice/assets/1572885/a900ce51-bd1f-4eec-82bf-45805d2b8aac)

## Expected behavior
Idea: Show "publication type" instead?

# Additional context
See confused researcher: https://otrsdoza.ugent.be/otrs/index.pl?Action=AgentTicketZoom;TicketID=3426
 
## Risk classification – Consequences
*What's the impact?*
3: Marginal, people get confused

**Comment by nicolasfranck** [comment]
@mietcls what you are showing is the detail page on lib.ugent.be, not on biblio.ugent.be.

Your example seems to correct now on lib.ugent.be

Found another example: https://lib.ugent.be/en/catalog/pug01:01H618ZYT6NBV3JY5P5C2EPVK1?i=0&q=01H618ZYT6NBV3JY5P5C2EPVK1, which is "journal article" on biblio.ugent.be.

**Comment by nicolasfranck** [comment]
The mapping from biblio to lib.ugent.be is done by lludss-import, and seems to focus on the field `classification` (which is mapped to marc field `920a` which becomes `type` in lib.ugent.be:

https://github.ugent.be/Universiteitsbibliotheek/lludss-import/blob/master/lib/Catmandu/Importer/Biblio.pm#L263

@phochste ?

**Comment by mietcls** [comment]
@nicolasfranck yes this about it's shown in lib. Whenever a publication is undefined, it shows as "misc" on lib – which is "technically" incorrect. To be refined what the outcome should be. Not urgent at all (hence "to refine" status).

**Comment by nics** [comment]
@nicolasfranck can you update https://github.ugent.be/Universiteitsbibliotheek/lludss-import/blob/master/lib/Catmandu/Importer/Biblio.pm to use type instead of classification to deteremine 920a?

(should be a lluddss-import issue)

**Comment by nicolasfranck** [comment]
@nics I'd rather have @phochste explain why it was done the way it was done. Maybe there were good reasons

**Comment by phochste** [comment]
The reasoning at the time was 'classification' is the officially type of the article as approved by UGent. The 'type' is just a choice by the person that created the record. 

Also in discussions how to map data the 'classification' is more well known. The question was for lib 'put all A1-s in such a way and P in that way.

Also records based on classification had other processing rules in bellow. E.g. B3-s are known to to have often full-text that doesn't really contain the full-text of the thing that is described in the record.

**Comment by nicolasfranck** [comment]
@mietcls I guess this is a bit unclear. Something to discuss, and move to discussions? Implementation is unclear anyway

**Comment by mietcls** [comment]
Yes! Thanks @nicolasfranck & @phochste
